app/controllers/concerns/phone_confirmation.rb

```diff
@@ -1,26 +1,10 @@
 module PhoneConfirmation
-  def prompt_to_confirm_phone(phone:, otp_method: nil, context: 'confirmation')
+  def prompt_to_confirm_phone(phone:, context: 'confirmation')
     user_session[:unconfirmed_phone] = phone
     user_session[:context] = context
-    # If the user selected delivery method, the code is sent and user is
-    # prompted to confirm.
-    prompt_to_choose_delivery_method and return unless otp_method
 
     redirect_to otp_send_path(
-      otp_delivery_selection_form: { otp_method: otp_method }
+      otp_delivery_selection_form: { otp_method: current_user.otp_delivery_preference }
```

Comment:
Ah. I misunderstood. You were asking if we should set a default. I thought you were complaining that we have a default set, because [we do](https://github.com/18F/identity-idp/blob/master/db/schema.rb#L134). It's set to `sms` by default.